helios-services/src/main/java/com/spotify/helios/agent/AgentConfig.java

```diff
@@ -369,4 +371,13 @@ public AgentConfig setConnectionPoolSize(final int connectionPoolSize) {
     this.connectionPoolSize = connectionPoolSize;
     return this;
   }
+
+  public File getGoogleCloudAccountCredentials() {
```

Comment:
Good point, although this is kind of difficult with the structure of ContainerRegistryAuthSupplier that is currently in docker-client. It would need to be refactored too, to allow passing the GoogleCredentials in. I think this would be a better design, but I'll probably hold off on doing so until there is a need for Helios to use the ADC.